src/app/main/component/comments/services/comments.service.spec.ts

```diff
@@ -61,12 +61,12 @@ describe('CommentsService', () => {
     req.flush(commentBody);
   });
 
-  it('should set id 0 if nothing wasn\'t send in parameters', () => {
+  it(`should set id 0 if nothing wasn't send in parameters`, () => {
```

Comment:
use double-quote and single-quote instead of backticks

Reply:
I got an error when the linter was checking the code that I can't use double quotes